src-tauri/src/commands/binaries.rs

```diff
@@ -570,6 +570,8 @@ pub async fn open_repl(
       &bin_ext("goalc"),
       "--proj-path",
       &data_folder.to_string_lossy(),
+      "--game",
+      &game_name,
     ])
```

Comment:
you'll have to do a check like the other functions and conditionally add the flag, since older tooling versions will fail over "unexpected arg"
https://github.com/open-goal/launcher/blob/fcf48ccf84f708fd315ea4d93ec2586ad7fe8f89/src-tauri/src/commands/binaries.rs#L303-L307

Although.,..this is goalc.  It'll be a different version...we'll have to see when we added the `--game` arg to goalc first.

Reply:
Kind of a hack but we could only do it for jak 2? I assume `goalc` has supported it since before the jak 2 release?

Reply:
yeah that should be a fine temporary hack, can be replaced with a proper version check eventually

Reply:
yeah that should be a fine temporary hack, can be replaced with a proper version check eventually